tilewe/src/ctilewemodule.c

```diff
@@ -35,7 +35,7 @@ static int Board_init(BoardObject* self, PyObject* args, PyObject* kwds)
 
 static PyObject* Board_CurrentPlayer(BoardObject* self, void* closure) 
 {
-    return PyBool_FromLong(self->Board.CurTurn); 
+    return PyLong_FromLong(self->Board.CurTurn); 
```

Comment:
fix for broken tournaments, it was only ever returning player 0 or player 1 (int as bool): player 0 when it was player 0's turn and player 1 when it was player 1, 2, or 3's turn